CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 #### Enhancements
 
+* VerticalWhitespaceClosingBracesRule: Add specialized configuration.  
```

Comment:
Please be more descriptive here. I suggest something like:
```suggestion
* Add new option `only_enforce_before_trivial_lines` to 
  `vertical_whitespace_closing_braces` rule. It allows to
  restrict the rule to simple closing braces, while an empty
  line before closing braces followed by non-trivial code
  (if-statements, parameters, ...) will be accepted.  
```

Reply:
Done.